docs-en/chapter_array_and_linkedlist/list.md

```diff
@@ -857,13 +857,13 @@ After sorting the list, we can use algorithms often tested in array-related algo
 
 ## List Implementation
 
-Many programming languages have built-in lists, such as Java, C++, Python, etc. Their implementations are quite complex, with very meticulous settings for parameters such as initial capacity and expansion multiplier. Interested readers can refer to the source code for learning.
+Many programming languages come with lists, including Java, C++, Python, etc. Their implementations tend to be intricate, featuring carefully considered settings for various parameters, like initial capacity and expansion factors. Readers who are curious can delve into the source code for further learning.
```

Comment:
I'm sorry my bad. In my previous comment, what I wanted to say was that I suggest removing the word 'structure' only, and keeping 'built-in'. Very sorry for the confusion.

"lists" -> "built-in lists"

Reply:
No problem, it's just a minor change. I have submitted a new commit, both of them make sense. :)